src/javascript/JContent/ContentRoute/ContentLayout/Upload/UploadItem/UploadItem.jsx

```diff
@@ -143,9 +141,9 @@ export class UploadItem extends React.Component {
             setTimeout(() => {
                 this.props.uploadFile(upload);
                 this.props.updateUploadsStatus();
-                const tagsList = updateReturnObj?.data?.jcr?.addNode?.awsRecognition?.tagImageSync;
-                if (typeof tagsList !== 'undefined' && tagsList !== null) {
-                    this.setState({numTags: tagsList.length});
+                const tagsComponent = uploadReturnObj;
```

Comment:
maybe rather use a property of "uploadReturnObj" (`uploadReturnObj.component`), if we need to pass other data later.

Reply:
Yup OK sounds good Updated